docs/UserGuide.md

```diff
@@ -427,13 +446,13 @@ Example:
 ## FAQ
 
 **Q**: How do I save changes?
-**A**: Athena's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-
-**Q**: Where are the save files that Athena uses?<br>
-**A**: The default save directory is in `{Athena home directory}/data`.
+**A**: Athena's data is saved in the hard disk automatically after any command that changes the data. There is no need to manually save the data.
 
 **Q**: What are the save files that Athena uses?<br>
-**A**: Athena uses three separate save files. They are `addressbook.json`, `tagtree.json` and `calendar.json`.
+**A**: Athena uses three separate save files. They are `addressbook.json`, `calendar.json` and `tagtree.json`.
+
+**Q**: Where are the save files that Athena uses stored?<br>
+**A**: The default save directory is in `{Athena home directory}/data`.
```

Comment:
I would argue that where the save files are located should come before what the save files are.  Would like to hear your thoughts on this.